CMakeGlobals.txt

```diff
@@ -64,6 +64,8 @@ else()
    set(RSTUDIO_PRO_BUILD 0)
 endif()
 
+set(RSTUDIO_DEFAULT_LOG_PATH "/var/log/rstudio/rstudio-server")
```

Comment:
Should we guard this by something like `if(NOT DEFINED <...>)`? (in case someone needed to override this from the command line invocation of CMake)

Reply:
Good idea. I'll make that change.